### DISCUSSION TITLE
Unable to click on the link of some documents

### DISCUSSION THREAD

**Discussion by t9NVYj68** [YetiForceCompany/YetiForceCRM#15400] [How to] [answered]
Hello,
On some documents I imported, I'm not able to click on the links (on the file name) to download them
Working example:
![image](https://user-images.githubusercontent.com/34187647/139816720-da3cd746-f071-4809-aaa1-8aac337ad925.png)

Example that doesn't work:
![image](https://user-images.githubusercontent.com/34187647/139816854-469d477a-7269-483d-9a43-6b4a478bfa7e.png)

Do you have any idea ?
Thanks

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/10198654/139846749-5f18c78b-2917-4a0e-ba8c-67fdf837fc43.png)

**Reply by t9NVYj68** [reply]
Thank you very much for the quick feedback
